interfaces/handler/user_test.go

```diff
@@ -812,6 +812,91 @@ func TestUserHandler_PatchAccount(t *testing.T) {
 	}
 }
 
+func TestUserHandler_DeleteAccount(t *testing.T) {
```

Comment:
表記ゆれがすごくて申し訳ないんですが:pray:
```suggestion
func TestUserHandler_DeleteUserAccount(t *testing.T) {
```